ballerina/tests/generated_client.bal

```diff
@@ -6,6 +6,7 @@ const BUILDING = "building";
 const WORKSPACE = "workspace";
 const DEPARTMENT = "department";
 const EMPLOYEE = "employee";
+const ORDERITEM = "orderItem";
```

Comment:
```suggestion
const ORDER_ITEM = "orderitem";
```

Reply:
Fixed in [f9dd2cb](https://github.com/ballerina-platform/module-ballerina-persist/pull/105/commits/f9dd2cbe2b6d8409c511f40bf1fd105b7b99858d)